data/start-cosmic

```diff
@@ -25,16 +25,12 @@ fi
 
 export XDG_CURRENT_DESKTOP="${XDG_CURRENT_DESKTOP:=pop:COSMIC}"
 export XDG_SESSION_TYPE="${XDG_SESSION_TYPE:=wayland}"
-export XCURSOR_THEME="${XCURSOR_THEME:=Adwaita}"
+export XCURSOR_THEME="${XCURSOR_THEME:=Pop}"
 export _JAVA_AWT_WM_NONREPARENTING=1
-
-# experimental X support in cosmic, but still prefer wayland
-if [[ ! -z "${COSMIC_X}" ]]; then
-    export DISPLAY=:0
-    export GDK_BACKEND=wayland,x11
-    export MOZ_ENABLE_WAYLAND=1
-    export COSMIC_BACKEND=kms
-fi
+export DISPLAY=:0
```

Comment:
We don't have to explicitly set the `DISPLAY`, that might potentially break multiple running sessions.

cosmic-comp will send the variable just like the `WAYLAND_DISPLAY` later to the session: https://github.com/pop-os/cosmic-comp/blob/xwayland/src/session.rs#L81

(And also imports it into systemd's user environment.)

Reply:
Ahh ok, I added it because it seemed like the launcher and other shell apps weren't getting DISPLAY maybe. With DISPLAY set this way i can start steam via launcher, but otherwise i wasn't able to. I'll have to look into that some more, but I'll remove it when i figure it out.

Reply:
Ya it seems like cosmic-session is only receiving WAYLAND_DISPLAY for now. I guess the xwayland_state might not be fully set up yet?

Reply:
Should be fixed with https://github.com/pop-os/cosmic-comp/commit/4940d0823d9ff26c48c0895e4dc6dbe12949738c

Reply:
Yup!